bambi/tests/test_built_models.py

```diff
@@ -338,6 +338,40 @@ def test_laplace_regression():
     bmb_model.fit()
 
 
+def test_logistic_regression_numpyro():
+    y = pd.Series(np.random.choice(["a", "b"], 50), dtype="category")
+    data = pd.DataFrame({"y": y, "x": np.random.normal(size=50)})
+    model = Model("y ~ x", data, family="bernoulli")
+    model.fit(method="nuts_numpyro", chain_method="vectorized")
+
+
+def test_logistic_regression_blackjax():
+    y = pd.Series(np.random.choice(["a", "b"], 50), dtype="category")
+    data = pd.DataFrame({"y": y, "x": np.random.normal(size=50)})
+    model = Model("y ~ x", data, family="bernoulli")
+    model.fit(method="nuts_blackjax", chain_method="vectorized")
+
+
+def test_regression_blackjax():
```

Comment:
Nit: These two tests `test_regression_blackjax` and  `test_regression_nunpyro` could be parameterized to reduce amount of code that needs to be read or maintained

Reply:
I agree with @canyon289 here, but if you want @markgoodhead you can open an issue fix this later.